### ISSUE TITLE
Queue enhancements

### ISSUE THREAD

**Issue by noah-schechter** [milesmcc/atlos#52]
Queue is looking fantastic. I'm looking to address two related issues here: 1) indicating to the researcher which media is the top priority and 2) deciding exactly what media should be featured in the queue.

**Comment by noah-schechter** [comment]
On #1: Because cards are organized in the same orientation as All Media, I'm worried researchers first arriving to the Queue won't have a clear, unambiguous indication of what media they should help catalog. I'm thinking using a vertical stack of cards for each Flag ("Needs Confirmation," "Help Needed") will help with this—a researcher will view the top card as the top priority. Jira reference:

<img width="1009" alt="Screen Shot 2022-05-07 at 22 42 06" src="https://user-images.githubusercontent.com/100018299/167283663-1f294b5b-160f-44f4-90bf-dab6f62405b7.png">

**Comment by noah-schechter** [comment]
2) I think there are three categories where user action is helpful:* "Needs Geolocation, [this one is inferred from an unset geolocation attribute]" "Help Needed," and "Needs Confirmation." 

*Caveat here: if we're using the same Queue page for community coordinators, a fourth category, "Community Confirmed," would also be helpful.

**Comment by noah-schechter** [comment]
Curious to get your thoughts on both.

**Comment by milesmcc** [comment]
Yeah, perhaps queue has two parts: `In Progress` and `Finalized` (or something). `In Progress` has `Help Needed`, `Needs Confirmation`, and `Needs Geolocation`, while `Finalized` has maybe `Community Confirmed` and `Closed`.

**Comment by noah-schechter** [comment]
Conclusion from conversation: Three columns one one page: Needs Geolocation, Help Needed, and Needs Confirmation. We'll build an organizational partner queue when we better understand their needs.
